scripts/test-huggingface-embedding.py

```
@@ -0,0 +1,65 @@
+            # Generate embedding
+            with torch.no_grad():
+                outputs = model(**inputs)
+                # Use mean pooling for sentence-level embedding
+                embeddings = outputs.last_hidden_state.mean(dim=1)
+
+            logger.info(f"Embedding shape: {embeddings.shape}")
+            logger.info(f"Embedding sample (first 5 values): {embeddings[0][:5].tolist()}")
+            print("-" * 50)
+
+        logger.info("✅ All tests completed successfully!")
+        return True
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)